### DISCUSSION TITLE
Issue with start

### DISCUSSION THREAD

**Discussion by aspraveen** [adonisjs/core#1755] [Help]
Any Idea what may be the cause of this issue?

Note: I just did the update to the latest beta version

whenever I run 

node ace serve 

```
TypeError: commandInstance.exec is not a function


1 Kernel.runCommand
  
node_modules/@adonisjs/core/node_modules/@adonisjs/ace/build/src/Kernel/index.js:238

2 App.handle
 node_modules/@adonisjs/core/build/src/Ignitor/Ace/App/index.js:301

3 Ace.handle
  node_modules/@adonisjs/core/build/src/Ignitor/Ace/index.js:3
```

**Comment by laxadev** [comment]
I have exactly the same problem. I updated to the latest version 5.0.1 but I had to downgrade to 5.0.0-preview-rc-1.12

**Comment by dmcardoso** [comment]
Same here.

**Comment by thetutlage** [comment]
You don't update to differently tagged versions (next in this case) until they are announced and have release notes.
